src/types/publicationFieldNames.ts

```diff
@@ -50,49 +34,32 @@ export enum DegreeType {
 }
 
 // Enums representing name of fields used by Formik
-export enum ReferenceFieldNames {
-  PUBLICATION_TYPE = 'entityDescription.publicationType',
+export enum PublicationFieldNames {
+  ARTICLE_NUMBER = 'entityDescription.reference.publicationInstance.articleNumber',
   DOI = 'entityDescription.reference.doi',
-}
-
-export enum JournalArticleFieldNames {
-  SUB_TYPE = 'entityDescription.publicationSubtype',
-  PUBLISHER = 'entityDescription.reference.publicationContext',
-  VOLUME = 'entityDescription.reference.publicationInstance.volume',
+  ISBN = 'entityDescription.isbn',
   ISSUE = 'entityDescription.reference.publicationInstance.issue',
+  NUMBER_OF_PAGES = 'entityDescription.numberOfPages',
   PAGES_FROM = 'entityDescription.reference.publicationInstance.pages.begin',
   PAGES_TO = 'entityDescription.reference.publicationInstance.pages.end',
-  ARTICLE_NUMBER = 'entityDescription.reference.publicationInstance.articleNumber',
-  PEER_REVIEW = 'entityDescription.reference.publicationInstance.peerReviewed',
-}
-
-export enum BookFieldNames {
-  SUB_TYPE = 'entityDescription.publicationSubtype',
-  PUBLISHER = 'entityDescription.reference.publicationContext',
-  ISBN = 'entityDescription.isbn',
   PEER_REVIEW = 'entityDescription.reference.publicationInstance.peerReviewed',
-  TEXT_BOOK = 'entityDescription.textBook',
-  NUMBER_OF_PAGES = 'entityDescription.numberOfPages',
-  SERIES = 'entityDescription.series',
-}
-
-export enum ReportFieldNames {
-  SUB_TYPE = 'entityDescription.publicationSubtype',
+  PUBLICATION_TYPE = 'entityDescription.publicationType',
   PUBLISHER = 'entityDescription.reference.publicationContext',
-  ISBN = 'entityDescription.isbn',
-  NUMBER_OF_PAGES = 'entityDescription.numberOfPages',
   SERIES = 'entityDescription.series',
-}
-
-export enum DegreeFieldNames {
   SUB_TYPE = 'entityDescription.publicationSubtype',
-  PUBLISHER = 'entityDescription.reference.publicationContext',
-  SPECIALISATION = 'entityDescription.numberOfPages',
-  SERIES = 'entityDescription.series',
+  TEXT_BOOK = 'entityDescription.textBook',
+  VOLUME = 'entityDescription.reference.publicationInstance.volume',
 }
 
-export enum ChapterFieldNames {
-  PUBLISHER = 'entityDescription.reference.publicationContext',
-  PAGES_FROM = 'entityDescription.reference.publicationInstance.pages.begin',
-  PAGES_TO = 'entityDescription.reference.publicationInstance.pages.end',
+export enum DescriptionFieldNames {
```

Comment:
klarer ikke helt å bestemme meg for hva jeg syns er best naming/oppdeling av disse... 
Feltene i `DescriptionFieldNames` er jo like logisk tilhørende `PublicationFieldNames` som feltene som ligger der imo.
kanskje burde `PublicationFieldNames` renames til `ReferenceFieldNames`, så har vi de fremdeles splittet per tab?

Reply:
godt poeng! støtter å bruke `ReferenceFieldNames`